### PR TITLE
Adds plasmaman screams

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
Adds sounds Farquaar made and posted on the forums as screams for plasmemes https://tgstation13.org/phpBB/viewtopic.php?p=634202#p634202

After further incoherent screaming in the old folk's house Dragomagol gave me permission to make everyone's life worse by making plasmemes noisy
image

## Why It's Good For The Game
More feedback for when you inconvenience/crush/injure/maim plasmemes is funny
https://streamable.com/ebw9ld
Such sweet cacophony
 
## Changelog
:cl: Farquaar and ArcaneDefence
add: Plasmamen can scream now
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
YEEEEEEEEEEEAH

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 1]
```diff
@@ -176,6 +176,13 @@
 		// Do normal metabolism
 		return FALSE
 
+/datum/species/plasmaman/get_scream_sound(mob/living/carbon/human)
+		return pick(
+			'sound/voice/plasmaman/plasmeme_scream_1.ogg',
+			'sound/voice/plasmaman/plasmeme_scream_2.ogg',
+			'sound/voice/plasmaman/plasmeme_scream_3.ogg',
+		)
```
Comment (dragomagol) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm:184]:
```suggestion
/datum/species/plasmaman/get_scream_sound(mob/living/carbon/human)
	return pick(
		'sound/voice/plasmaman/plasmeme_scream_1.ogg',
		'sound/voice/plasmaman/plasmeme_scream_2.ogg',
		'sound/voice/plasmaman/plasmeme_scream_3.ogg',
	)
```

**Review by dragomagol — APPROVED** [review]

**Comment by VladinXXV** [comment]
can you give skeletons and bone golems the file(s) `sound/magic/RATTLEMEBONES.ogg` and/or `sound/magic/RATTLEMEBONES2.ogg`  as their screams while you're at it? it is absolutely tragic that they do not use this already.

**Comment by Sealed101** [comment]
very cool but not as cool as revenant screams

we most probably can't use revenant screams so this is cool

**Comment by ArcaneDefence** [comment]
> can you give skeletons and bone golems the file(s) `sound/magic/RATTLEMEBONES.ogg` and/or `sound/magic/RATTLEMEBONES2.ogg` as their screams while you're at it? it is absolutely tragic that they do not use this already.

These are violently too intrusive for normal screams imo, so I'm not interested in even trying to convince people

**Comment by Boopideedoo** [comment]
MERGE IT THIS IS SOMETHING WE ABSOLUTELY NEED!!!!

**Comment by Farquaar** [comment]
Glorious.

**Pull request merged**